docs/reference/cluster/stats.asciidoc

```diff
@@ -38,6 +38,278 @@ include::{docdir}/rest-api/common-parms.asciidoc[tag=flat-settings]
+Unique identifier for the cluster.
+
+`timestamp`::
+(https://en.wikipedia.org/wiki/Unix_time[Unix timestamp])
+Last time the cluster statistics were refreshed.
+
+`status`::
+(string)
+Health status of the cluster based on the state of its primary and replica
+shards. Returned values include:
+
+  `green`:::
+  All primary shard and replica shards are assigned. There are at least two
+  copies of all shards.
+
+  `yellow`:::
+  All primary shards are assigned, but one or more replica shards are
+  unassigned. If a node in the cluster fails, data could be lost.
+
+  `red`:::
+  One or more primary shards are unassigned. This can occur briefly during
+  cluster startup as primary shards are assigned during cluster startup. At
+  other times, this status may indicate data loss.
+
+`indices.count`::
+(integer)
+Total number of indices in the cluster.
+
+`indices.shards.total`::
+(integer)
+Total number of shards for all indices.
+
+`indices.shards.primaries`::
+(integer)
+Number of primary shards.
+
+`indices.shards.replication`::
+(integer)
+Number of replica shards.
+
+`indices.shards.index.shards.min`::
+(integer)
+Lowest historic number of shards for the cluster.
+
+`indices.shards.index.shards.max`::
+(integer)
+Highest historic number of shards for the cluster.
+
+`indices.shards.index.shards.avg`::
+(integer)
+Average number of shards for the cluster.
+
+`indices.shards.index.primaries.min`::
+(integer)
+Lowest historic number of primary shards for the cluster.
+
+`indices.shards.index.primaries.max`::
+(integer)
+Highest historic number of primary shards for the cluster.
+
+`indices.shards.index.primaries.avg`::
+(integer)
+Average number of primary shards for the cluster.
+
+`indices.shards.index.replication.min`::
+(integer)
+Lowest historic number of replica shards for the cluster.
+
+`indices.shards.index.replication.max`::
+(integer)
+Highest historic number of replica shards for the cluster.
+
+`indices.shards.index.replication.avg`::
+(integer)
+Average number of replica shards for the cluster.
+
+`indices.docs.count`::
+(integer)
+Number of non-deleted documents in the cluster.
++
+This number is based on documents in Lucene segments and may include documents
+from nested fields.
+
+`indices.docs.deleted`::
+(integer)
+Number of deleted documents in the cluster.
++
+This number is based on documents in Lucene segments. {es} reclaims the disk
+space of deleted Lucene documents when a segment is merged.
+
+`indices.store.size`::
+(<<byte-units, byte units>>)
+Size of the indices store.
+
+`indices.store.size_in_bytes`::
+(integer)
+Size, in bytes, of the store.
+
+`indices.fielddata.memory_size`::
+(<<byte-units, byte units>>)
+Memory used for fielddata cache.
+
+`indices.fielddata.memory_size_in_bytes`::
+(integer)
+Memory, in bytes, used for fielddata cache.
+
+`indices.fielddata.evictions`::
+(integer)
+Number of fielddata evictions.
+
+`indices.query_cache.memory_size`::
+(<<byte-units, byte units>>)
+Memory used for query cache.
```

Comment:
Total memory used for the query cache on all shards (primaries and replicas) across all selected nodes.

Reply:
Fixed with 62b05d32cd6.